### ISSUE TITLE
Game loops loading screen when entering infinite climb room a second time

### ISSUE THREAD

**Issue by Rummskartoffel** [renaudbedard/fez-1.12-issues#177]
[[2017-05-04] Debug Log #4.txt](https://github.com/renaudbedard/fez-1.12-issues/files/977464/2017-05-04.Debug.Log.4.txt)
When entering the infinite climb room, leaving, and trying to re-enter, the game shows an infinite loading screen instead. The only way out of this is force stopping the game.

**Comment by wencelf** [comment]
Did you find a solution for this? I have the same problem and cannot seem to find a solution.

**Comment by Rummskartoffel** [comment]
I haven't played the game in quite a while, but if I remember correctly, after killing the game and restarting, the room works again. So you have to avoid re-entering the room, and close the game in between attempts if you have to. Judging from your comment, this has yet to be fixed, which is up to the developers.

**Comment by Radu13** [comment]
Wrong, the solution is to enable single thread mode, load/enter that room, grab the anticube, and turn single thread mode off again if not wanted.
Hopefully someone fill find this helpful, this bug really should be fixed!

**Comment by Anutrix** [comment]
Faced the same issue.
